### PR TITLE
feat(authorizenet): map errors in magento driver

### PR DESCRIPTION
**Pull request by griest024**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

The Magento anet driver only throws generic errors.

## What is the new behavior?
The Magento anet driver throws very specific errors.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

File: libs/authorizenet/driver/magento/src/authorize-net.service.ts [hunk 1]
```diff
@@ -22,7 +22,10 @@ export class DaffMagentoAuthorizeNetService implements DaffAuthorizeNetService {
 		return new Observable(observer =>
 			Accept.dispatchData(transformMagentoAuthorizeNetRequest(paymentTokenRequest, this.config), (response: AuthorizeNetResponse) => {
 				if (response.messages.resultCode === 'Error') {
-					throw new Error(response.messages.message[0].code + ':' + response.messages.message[0].text);
+          const MappedError = DAFF_AUTHORIZE_NET_ERROR_CODE_MAP[response.messages.message[0].code];
```
Comment (damienwebdev) [inline libs/authorizenet/driver/magento/src/authorize-net.service.ts:25]: Needs a test covering at least one case.

**Review by damienwebdev — CHANGES_REQUESTED** [review]

File: libs/authorizenet/driver/magento/src/authorize-net.service.ts (hunk 1 above)
Comment (griest024) [reply libs/authorizenet/driver/magento/src/authorize-net.service.ts:25]: The driver service doesn't have a unit test, I'll add one

Comment (griest024) [reply libs/authorizenet/driver/magento/src/authorize-net.service.ts:25]: I added a test but messing with the `Accept` global doesn't seem to work in `beforeEach` so I added it each `it`.

File: libs/authorizenet/src/services/accept-js-loading.service.ts [hunk 2]
```diff
@@ -18,6 +18,8 @@ export class DaffAcceptJsLoadingService {
 	) {}
 
 	load(): void {
+    console.log(typeof Accept);
```
Comment (damienwebdev) [inline libs/authorizenet/src/services/accept-js-loading.service.ts:21]: log

**Review by damienwebdev — CHANGES_REQUESTED** [review]
minor comment, but otherwise lgtm.

File: libs/authorizenet/src/services/accept-js-loading.service.ts (hunk 2 above)
Comment (griest024) [reply libs/authorizenet/src/services/accept-js-loading.service.ts]: removed

**Review by damienwebdev — APPROVED** [review]
lgtm

**Pull request merged**